cloud/pkg/cloudhub/cloudhub.go

```diff
@@ -1,6 +1,7 @@
 package cloudhub
 
 import (
+	"github.com/kubeedge/kubeedge/cloud/pkg/common/modules"
```

Comment:
Call `goimport -w -local "github.com/kubeedge/kubeedge"` to sort.

Reply:
done. thx.